adapters/remote-agent/src/nemo_fabric_adapters/remote_agent/adapter.py

```
@@ -0,0 +1,405 @@
+        if item.get("type") != "message":
+            continue
+        parts = item.get("content", [])
+        text = "".join(
+            part["text"] for part in parts if part.get("type") == "output_text"
+        )
+        if text:
+            return text
+    raise RemoteAgentError("remote agent response did not include output text")
+
+
+async def _sse_events(response: httpx.Response):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_sse_events`

(ANN202)